<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HexaWindow</class>
 <widget class="QMainWindow" name="HexaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1329</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_workspace">
       <attribute name="title">
        <string>Workspace</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="grid_workspace">
          <item row="14" column="1">
           <widget class="QLabel" name="lbl_duty">
            <property name="text">
             <string>Duty:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QCheckBox" name="checkBox_LA1_Op">
            <property name="text">
             <string>LA1 Operational</string>
            </property>
           </widget>
          </item>
          <item row="13" column="0">
           <widget class="QCheckBox" name="togPosVelStreamData">
            <property name="text">
             <string>Stream Position &amp; Velocity Data</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QCheckBox" name="checkBox_LA2_Op">
            <property name="text">
             <string>LA2 Operational</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QCheckBox" name="checkBox_LA4_Op">
            <property name="text">
             <string>LA4 Operational</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QCheckBox" name="checkBox_LA0_Op">
            <property name="text">
             <string>LA0 Operational</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QCheckBox" name="checkBox_LA3_Op">
            <property name="text">
             <string>LA3 Operational</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QListWidget" name="listView_WorkspaceSelect"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Linear Actuator - Current Workspace</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QCheckBox" name="checkBox_LA5_Op">
            <property name="text">
             <string>LA5 Operational</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="comPortSelect"/>
          </item>
          <item row="14" column="0">
           <widget class="QCheckBox" name="togSDKmode">
            <property name="text">
             <string>SDK Mode</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" colspan="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>5000</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="3" rowspan="14" colspan="2">
           <layout class="QVBoxLayout" name="vertGraphLayout">
            <item>
             <widget class="PlotWidget" name="widget" native="true"/>
            </item>
            <item>
             <widget class="PlotWidget" name="myWidget" native="true"/>
            </item>
           </layout>
          </item>
          <item row="14" column="2">
           <widget class="QLabel" name="lbl_dutyValue">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="btnTimeBasedDemo">
              <property name="text">
               <string>Time Based Demo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnTimeBasedOpen">
              <property name="text">
               <string>Time Based Open</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnTimeBasedClosed">
              <property name="text">
               <string>Time Based Closed</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QListWidget" name="listView_ControllerMode"/>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Controller Mode</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="enterCommand"/>
            </item>
            <item>
             <widget class="QPushButton" name="sendCommand">
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QTextEdit" name="historyCommand">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QLabel" name="lbl_posKp">
            <property name="text">
             <string>Pos Kp:</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <widget class="QLabel" name="lbl_velKp">
            <property name="text">
             <string>Vel Kp:</string>
            </property>
           </widget>
          </item>
          <item row="13" column="1">
           <widget class="QLabel" name="lbl_velKi">
            <property name="text">
             <string>Vel Ki:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>1000</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="11" column="2">
           <widget class="QDoubleSpinBox" name="spinBox_posKp">
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="12" column="2">
           <widget class="QDoubleSpinBox" name="spinBox_velKp">
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="13" column="2">
           <widget class="QDoubleSpinBox" name="spinBox_velKi">
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_collective">
       <attribute name="title">
        <string>Collective</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="grid_collective">
          <item row="0" column="3">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10000</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lblSlide_LA3">
            <property name="text">
             <string>LA3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="btnHome_LA2">
            <property name="text">
             <string>Home</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblSlide_LA1">
            <property name="text">
             <string>LA1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblSlide_LA2">
            <property name="text">
             <string>LA2</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lblSlide_LA4">
            <property name="text">
             <string>LA4</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QPushButton" name="btnHome_LA5">
            <property name="text">
             <string>Home</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSlider" name="horizontalSlider_LA5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="lblSlide_LA5">
            <property name="text">
             <string>LA5</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="btnHome_LA3">
            <property name="text">
             <string>Home</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSlider" name="horizontalSlider_LA3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QPushButton" name="btnHome_LA4">
            <property name="text">
             <string>Home</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSlider" name="horizontalSlider_LA4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="horizontalSlider_LA2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lblSlide_LA0">
            <property name="text">
             <string>LA0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="horizontalSlider_LA1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="horizontalSlider_LA0">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="btnCollectiveEnable">
              <property name="text">
               <string>Enable</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnHomeAll">
              <property name="text">
               <string>Home All</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnCollectiveDemo">
              <property name="text">
               <string>Demo</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btnHome_LA0">
            <property name="text">
             <string>Home</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btnHome_LA1">
            <property name="text">
             <string>Home</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>10000</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_firmware">
       <attribute name="title">
        <string>Firmware</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QGridLayout" name="grid_firmware">
          <item row="2" column="0" colspan="2">
           <widget class="QTextEdit" name="txt_compilerLog">
            <property name="font">
             <font>
              <family>Courier New</family>
              <pointsize>10</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(40, 40, 40);
color: rgb(200, 200, 200);
font-family: Courier New;
font-style: normal;
font-size: 10pt;
font-weight: Regular;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="lbl_compilerLog">
            <property name="text">
             <string>Compiler Log:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="inoFilePath"/>
            </item>
            <item>
             <widget class="QPushButton" name="inoLaunchPathDialog">
              <property name="text">
               <string>Select &quot;.ino&quot; Path</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="btn_compileOnly">
              <property name="text">
               <string>Compile Only</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_compileAndUpload">
              <property name="text">
               <string>Compile &amp;&amp; Upload</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_modelling">
       <attribute name="title">
        <string>Modelling</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="grid_modelling">
          <item row="1" column="2">
           <widget class="QPushButton" name="btnModelling_RunC">
            <property name="text">
             <string>Run C</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btnModelling_RunB">
            <property name="text">
             <string>Run B</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btnModelling_RunA">
            <property name="text">
             <string>Run A</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="PlotWidget" name="modellingPlot" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1329</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menugg">
    <property name="title">
     <string>Test</string>
    </property>
   </widget>
   <addaction name="menugg"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
